src/com/esotericsoftware/kryo/serializers/RecordSerializer.java

```diff
@@ -0,0 +1,216 @@
+/* Copyright (c) 2008-2020, Nathan Sweet
+ * Copyright (C) 2020, Oracle and/or its affiliates.
```

Comment:
Could you add yourself (and Oracle) as `@author` instead of adjusting the copyright headers. I'm not sure how @NathanSweet stands on this and it is difficult to get feedback from him at the moment.

Reply:
If possible I'd like to keep the copyright header as we are generally required to include it on any code we contribute.

Reply:
That's alright. In this case, we just need an OK from @NathanSweet or @magro.

Reply:
Having multiple copyright owners for Kryo doesn't hurt the project being FOSS, as all contributions must use the [contribution rules](https://github.com/EsotericSoftware/kryo/blob/master/CONTRIBUTING.md). However, it is preferable that we keep a single copyright owner because it simplifies enforcing the copyright. Otherwise all copyright owners must be involved and that can be difficult or impossible.